ape_safe/accounts.py

```diff
@@ -373,6 +373,60 @@ def create_safe_tx(self, txn: Optional[TransactionAPI] = None, **safe_tx_kwargs)
         }
         return self.safe_tx_def(**safe_tx)
 
+    def all_delegates(self) -> Iterator[AddressType]:
```

Comment:
generators are bad ux in interactive sessions.

change to `delegates` property and make it return a list or a dict of address->label

Reply:
we can have a method for the iterator such as `get_all_delegates()` and have the `delegates` property use the iterator to make a dict

Reply:
my other point is delegates is not a feature of a safe wallet, but rather a feature of a safe backend api, this it should not be in `SafeAccount` at all, but rather in `SafeClient`.

it's completely off-chain and only affects the backend accepting txs from non-owners.